apps/server/src/shared/domain/entity/dashboard.model.entity.ts

```diff
@@ -72,7 +72,7 @@ export class DashboardModelEntity extends BaseEntityWithTimestamps {
 		if (props.gridElements) this.gridElements.set(props.gridElements);
 	}
 
-	@OneToMany('DashboardGridElementModel', 'dashboard')
+	@OneToMany('DashboardGridElementModel', 'dashboard', { orphanRemoval: true })
```

Comment:
orphanRemoval makes sense, still this should be tested

Reply:
Right. Especially because that brings potential risk of data loss when referential integrity of the data is not given.